composer.json

```diff
@@ -8,20 +8,29 @@
   "require": {
     "php": "^8.0",
     "pimcore/pimcore": "^10.2",
-    "pimcore/data-hub": "^1.2",
-    "phpoffice/phpspreadsheet": "^1.1",
+    "pimcore/data-hub": "^1.2@dev",
+    "phpoffice/phpspreadsheet": "^1.15",
     "dragonmantank/cron-expression": "^3.1",
+    "nesbot/carbon": "^2.27",
     "league/flysystem": "^2.0",
     "league/flysystem-sftp": "^2.0",
     "league/flysystem-bundle" : "^2.0",
     "ext-json": "*",
     "ext-fileinfo": "*"
   },
+  "require-dev": {
+    "phpstan/phpstan": "^1.2"
+  },
   "autoload": {
     "psr-4": {
       "Pimcore\\Bundle\\DataImporterBundle\\": "src/"
     }
   },
+  "autoload-dev": {
```

Comment:
we could remove that too now, can't we?

Reply:
Yes, I removed it https://github.com/pimcore/data-importer/pull/141/commits/0e9658b8631158af5055042385f7e62f3fd9ad1b